error-handling.js

```diff
@@ -5,3 +5,15 @@ exports.customErrorHandling = (err, req, res, next) => {
 		next(err);
 	}
 };
+
+exports.psqlErrorHandling = (err, req, res, next) => {
+	if (err.code) {
+		switch (err.code) {
+			case '22P02':
+				res.status(400).send({ msg: 'Bad request' });
+				break;
+		}
+	} else {
+		console.log(err, '<-- logged err in psql err handling');
+	}
```

Comment:
You could maybe write a 500 handler and add it to the middleware chain. It could log any errors that make it past you r previous handlers and gracefully respond with a 500. (You can't really test this one easily though!)